xml_inscriptions/metadata_only/KY.Lou.SAM.L.1929.17.526.xml

```diff
@@ -107,18 +109,21 @@
       
         <body>
         <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+           <ab>
+              <lb n="1"/><persName><name type="praenomen" key="Gaius"><w part="I"><expan><abbr>C</abbr><ex>ai</ex></expan></w></name>
```

Comment:
As a rule, we don't normally use the `<w>` element at USEP.

Reply:
I should note, it's used correctly here; we just aren't in the practice of using it.